client/src/components/explorer/style.scss

```diff
@@ -1,13 +1,373 @@
+$c-explorer-bg: #4C4E4D;
+$c-explorer-secondary: #aaa;
+$c-explorer-easing: cubic-bezier(0.075, 0.820, 0.165, 1.000);
+
+.c-explorer * {
+    box-sizing: border-box;
+}
+
 .c-explorer {
-    width: 320px;
+    width: 100%;
     height: 500px;
+    background: $c-explorer-bg;
+    position: absolute;
+    overflow: hidden;
+}
+
+    .c-explorer--sidebar {
+        height: 100vh;
+        box-shadow: 2px 2px 5px rgba(0,0,0,0.2);
+        z-index: 150;
+    }
+
+.c-explorer__header {
+    border-bottom: solid 1px #676767;
+    overflow: hidden;
+    color: $c-explorer-secondary;
+}
+
+.c-explorer__trigger {
+    display: block;
+    padding: .5rem 1rem;
+    white-space: nowrap;
+    overflow: hidden;
+    width: 80%;
+    float: left;
+}
+
+.c-explorer__trigger--enabled {
+    cursor: pointer;
+
+    &:hover {
+        color: #fff;
+        background: rgba(0,0,0,0.2);
+    }
+}
+
+.c-explorer__filter {
+    float: right;
+    width: 50px;
+    margin-top: .5rem;
+}
+
+.c-filter {
+    display: inline-block;
+    vertical-align: middle;
+    padding: 0 .25em;
+    border: solid 1px rgba(255,255,255,0.1);
+    border-radius: 2px;
+    line-height: 1;
+    margin-left: .25rem;
+    cursor: pointer;
+    &:hover {
+        background: rgba(0,0,0,0.5);
+        border-color: rgba(0,0,0,0.5);
+        color: #fff;
+    }
+}
+
+.c-filter--active {
+    color: #fff;
+    border-color: rgba(255, 255, 255, .5);
+}
+
+
+.c-explorer__back {
+    margin-right: .25rem;
+    float: left;
+    margin-top: -1px;
+
+    &:hover {
+        color: #fff;
+    }
+
+    .icon {
+        line-height: 1;
+        display: inline-block;
+        font-size: 16px;
+    }
+}
+
+.c-explorer__title {
+    margin: 0;
+    color: #fff;
+}
+
+.c-explorer__loading {
+    color: #fff;
+    padding: 1rem;
+}
+
+.c-explorer__item {
+    padding: 1rem;
+    cursor: pointer;
+    border-bottom: solid 1px #676767;
+
+    &:last-child {
+        border-bottom: 0;
+    }
+}
+
+.c-explorer__placeholder {
+    padding: 1rem;
+    color: #fff;
+}
+
+.c-explorer__meta {
+    font-size: 12px;
+    color: $c-explorer-secondary;
+    margin-bottom: 0;
+}
+
+    // TODO Could be a utility class
+    .c-explorer__meta__type {
+        text-transform: capitalize;
+    }
+
+.c-explorer__item:hover {
+    background: rgba(0, 0, 0, 0.25);
+    color: #fff;
+}
+
+.c-explorer__see-more {
+    cursor: pointer;
+    padding: .5rem 1rem;
+    background: rgba(0,0,0,0.2);
+    color: #fff;
+
+    &:hover {
+        background: rgba(0,0,0,0.4);
+    }
+}
+
+
+.c-explorer__children {
+    display: inline-block;
+    border-radius: 50rem;
+    border: solid 1px #aaa;
+    color: #fff;
+    line-height: 1;
+    padding: .5em .3em .5em .5em;
+    float: right;
+    cursor: pointer;
+
+    &:hover {
+        background: rgba(0,0,0,0.5);
+    }
+
+    [aria-role='presentation'] {
```

Comment:
👍 I changed this to simply using the `visuallyhidden` class that already exists within the codebase.